lib/mps/linalg.jl

```diff
@@ -74,6 +74,8 @@ if VERSION < v"1.10.0-DEV.1365"
 # catch other functions that are called by LinearAlgebra's mul!
 LinearAlgebra.gemm_wrapper!(C::MtlMatrix, tA, tB, A::MtlMatrix, B::MtlMatrix, _add::MulAddMul) =
     LinearAlgebra.generic_matmatmul!(C, tA, tB, A, B, _add)
+LinearAlgebra.gemm_wrapper!(C::MtlMatrix{T}, tA, tB, A::MtlMatrix{T}, B::MtlMatrix{T}, _add::MulAddMul) where {T<:LinearAlgebra.BlasFloat} =
```

Comment:
Not sure if the test failure on v1.9 is due to this change. Should this perhaps be

```suggestion
LinearAlgebra.gemm_wrapper!(C::MtlMatrix{T}, tA, tB, A::MtlMatrix{T}, B::MtlMatrix{T}, _add::MulAddMul) where {T<:MtlFloat} =
```
?

Reply:
See #138 which should be fixed in Julia 1.9.2. Until then we'll have to live with intermittent and unrelated CI failures.